src/main/lrsql/input/actor.clj

```diff
@@ -61,6 +61,18 @@
    :actor-ifi    actor-ifi
    :actor-type   actor-type})
 
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Actor Delete
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(s/fdef delete-actor-input
+  :args (s/cat :actor-ifi (s/keys :req-un [:lrsql.spec.actor/actor-ifi]))
+  :ret  (s/keys :req-un [:lrsql.spec.actor/actor-ifi]))
+
+(defn delete-actor-input [actor-ifi-obj]
+  actor-ifi-obj)
```

Comment:
![image](https://github.com/yetanalytics/lrsql/assets/782173/81326366-1073-4df5-b43b-5930bee0ceba)

Reply:
i'm a people function!

Reply:
Yeah if it doesn't need an input function, I'd say remove it and just pass it along. It's been validated after all.

Reply:
Well, so there is something to discuss here. Typically this is where the actual input objects are placed into an input map. Why is that not the case here? I see in record its expecting an input map, so is it pre-formed before getting here or something?

Reply:
the check takes place at ingestion in `admin.interceptors.lrs-management`; the spec for the route (`{"actor-ifi": <actor-ifi>}`) is identical with the spec for the op (`{:actor-ifi <actor-ifi>}`)

Reply:
Yeah fair enough. Seems fine.